charts/konk/scripts/provision.sh

```diff
@@ -0,0 +1,17 @@
+kubeadm init phase certs all
```

Comment:
Exit on errors and spit out every command that is executed. Use /bin/sh if bash is not available in the pod
```suggestion
#!/bin/bash -xe

kubeadm init phase certs all
```

Reply:
Done